### ISSUE TITLE
ENS name is not shown in profile when selected as well as in [Chat settings] preview in Profile

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/status-desktop#7182]
### Description

1. when ens name is set, it should be shown in [Chat settings] bubble in Profile
2. when ens name is set, it should replace the display name in Profile

<img width="1512" alt="Screenshot 2022-08-30 at 10 39 04" src="https://user-images.githubusercontent.com/82375995/187378652-ff4a4509-cf37-40c0-b691-0764b38af564.png">

<img width="1512" alt="Screenshot 2022-08-30 at 10 40 17" src="https://user-images.githubusercontent.com/82375995/187378771-7b374a36-4803-4189-8396-f757ad8b1dbd.png">

**Comment by jrainville** [comment]
Yes that's right. I wonder if it's going to confuse users that they can no longer see or change their "old" display name, but I guess most people that have ENS set know what they are doing and are happy with just seeing the ENS name.
